runner/runners/invoke.go

```diff
@@ -248,9 +248,8 @@ func (inv *Invoker) run(cmd *runner.Command, id runner.RunID, abortCh chan struc
 		}
 		return status
 	case execer.FAILED:
-		// We treat failure here as a bad request since it's more likely a faulty cmd than something we did wrong.
-		return runner.BadRequestStatus(id, fmt.Errorf("error execing: %v", st.Error), runner.LogTags{JobID: cmd.JobID, TaskID: cmd.TaskID})
+		return runner.ErrorStatus(id, fmt.Errorf("error execing: %v", st.Error), runner.LogTags{JobID: cmd.JobID, TaskID: cmd.TaskID})
```

Comment:
Can we change runner.ErrorStatus() to runner.FailedStatus() to match execer.FAILED